api/functions/src/index.ts

```diff
@@ -2,12 +2,17 @@
 require('dotenv').config();
 import * as functions from 'firebase-functions';
 import { ConstructGraphQLServer } from './graphql/server';
+import { updateUsers } from './notifications/notificationService';
 import { ConstructFileDownloadServer } from './downloads/server';
 
 const USCentralRegion = functions.SUPPORTED_REGIONS[0];
 exports.graph = functions
   .region(USCentralRegion)
   .https.onRequest(ConstructGraphQLServer());
+// exports.notify = functions.pubsub
+//   .schedule('0 * * * *')
+//   .timeZone('Pacific/Auckland')
+//   .onRun(updateUsers);
```

Comment:
Do we need this commented out code?

Reply:
No, will uncomment it now.

Reply:
Wait, we need the code but it doesn't need to be commented out